docs/can-canjs/can-api.md

```diff
@@ -102,7 +101,7 @@ The following defines a `Todo` type with numerous property behaviors and
 a `toggleComplete` method.
```

Comment:
This explanation probably needs to be updated since the base example doesn't have "numerous property behaviors".